apps/test/unit/componentLibrary/ButtonTest.jsx

```diff
@@ -0,0 +1,247 @@
+// TODO: Once start working on Button component - uncomment the code and use it as a template for Button component tests
```

Comment:
nit: this makes it seem like the set of Checkbox tests is the template for Button tests. is that meant to be the case?

Reply:
yeah, that's intended. The test cases of checkbox are more or less simillar to what we'll need to test, but the signature of different elements is too different to write specific test without creating at least the base component version. So checkbox tests are here just to guide one through what should be done and what test patterns do we already have.